### ISSUE TITLE
check if there is pending migration on the server side

### ISSUE THREAD

**Issue by vpereira** [openSUSE/ansible-obs#22]
Today in the [run_migration role](https://github.com/openSUSE/ansible-obs/blob/master/roles/run_migrations/tasks/main.yml) we are running the command `obs_deploy pm` which checks in the code level if there is any pending migration. This command runs locally in our desktop. The question is if we shouldn't run (instead or additionally) the following commands in the remote server:

`rails db:migrate:status` and `rails data:migrate:status`.

**Comment by dmarcoux** [comment]
We should run this only on the remote server, because this is the only way to know the database state.

**Comment by vpereira** [comment]
> We should run this only on the remote server, because this is the only way to know the database state.

true, however today we are running the `obs_deploy pm`, which checks just in the code level. The question is if the server side solution should substitute it or add on top of that

**Comment by dmarcoux** [comment]
I would do it only on the server.
